cluster/sync.sh

```diff
@@ -2,35 +2,9 @@
 
 set -ex
 
-kubectl=./cluster/kubectl.sh
-
-function eventually {
-    timeout=$(( $KUBEVIRT_NUM_NODES * 10 ))
-    interval=5
-    cmd=$@
-    echo "Checking eventually $cmd"
-    while ! $cmd; do
-        sleep $interval
-        timeout=$(( $timeout - $interval ))
-        if [ $timeout -le 0 ]; then
-            return 1
-        fi
-    done
-}
+source ./cluster/sync-common.sh
```

Comment:
We are going to move deploy_operator to `sync-operator.sh` and since we are deployting handler and operator here we have to source `sync-operator.sh`

To do so we have to be sure that sync-operator.sh does not run stuff if we source it only if we run it.